code/datums/spells/rathens.dm

```diff
@@ -0,0 +1,57 @@
+/obj/effect/proc_holder/spell/aoe_turf/rathens
+	name = "Rathen's Secret"
+	desc = "Summons a powerful shockwave around you that tears the appendix and limbs off of enemies."
+	charge_max = 500
+	clothes_req = 1
+	invocation = "ARSE NATH!"
+	invocation_type = "shout"
+	range = 7
+	cooldown_min = 200
+	selection_type = "view"
+	action_icon_state = "superfart"
+
+/obj/effect/proc_holder/spell/aoe_turf/rathens/cast(list/targets)
+	var/mob/user = usr
+	var/list/impacted_mobs = list()
+	for(var/turf/T in targets)
```

Comment:
Also in general it's already too late to call `revert_cast()` inside of `cast()` since several things have already irreversibly happened, including the invocation, and starting the recharge.